source/index.html.md

```diff
@@ -583,6 +648,14 @@ Exactly one payment method must be used.
     A reference to an external object, such as an order number.
     <div class="type">Optional</div>
   </dd>
+
+  <!-- deprecated -->
+  <dt><strike>recurring</strike>
+    <span class="type">(true|false)</span>
+  </dt>
+  <dd>
+    Deprecated! Please use <code>series</code>.
+  </dd>
```

Comment:
I think we ought to update the examples here and there to use `series` rather than the deprecated `recurring`.

Reply:
E.g. in line 361.

Reply:
I understand that this doesn't make sense before the new parameters have stabilized.

Reply:
I'll fix this 🙂

Reply:
Was fixed by https://github.com/clearhaus/gateway-api-docs/pull/111/commits/87a8acf89466d7c1d3d8a1ea4f0cd244630e379c#diff-d30d91dbcd59693b69f9242acb4630c5R364